docs/APIReference-Editor.md

```diff
@@ -119,11 +119,20 @@ needed to observe spellcheck events are not fired in IE.
 
 Default is `false`.
 
-#### stripPastedStyles
+#### pasteSupport
 ```
-stripPastedStyles?: boolean
+stripPastedStyles?: Object
+```
+Optionally define which inline styles, block types and links should be retained on paste.
```

Comment:
We should go into more depth here to make the documentation easier to use. For example, we should clarify that the supported block types is the intersection between those defined in the block render map and those defined in this structure. We should provide an example, non-empty, value for this field as well so that users don't need to spelunk through the source to find how each field is being interpreted and used.